CHANGES.txt

```diff
@@ -84,6 +84,8 @@ Proposed Release Version: 0.9
     FALCON-1213 Base framework of the native scheduler(Pallavi Rao)
 
   IMPROVEMENTS
+    FALCON-1707 Minor doc update for tar package locations after FALCON-1751 (Deepak Barr)
```

Comment:
This change won't go into 0.9. So, can move this change up, under trunk section, please?